README.md

```diff
@@ -2,7 +2,14 @@
 
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)][code of conduct]
 
-This project is developed to be a foundational starter for creating your own NFT storefront on the Algorand blockchain. It is a monorepo that includes a Fastify API, a headless CMS (Directus), a front-end sample implementation (NextJS), and a shared schemas package to share Typescript interfaces and enums across the monorepo.
+This project is developed to be a foundational starter for creating your own NFT storefront on the Algorand blockchain. It is a monorepo that includes:
+
+- A [headless CMS](./apps/cms) (Directus)
+- A [back-end API](./services/api) (Fastify)
+- A [front-end](./apps/web) sample implementation (NextJS)
+- Shared Typescript [interfaces and enums](./packages/schemas)
+- [Terraform templates](./terraform) for setting up infrastructure on Google Cloud Platform
+- [Github Workflows](./.github/workflows) for linting, type-checking, building, dockerizing, and deploying
```

Comment:
This wasn't necessary, but it felt like it might be good to have a list that also includes the terraform/workflows stuff